docker-ee-for-windows/install.md

```diff
@@ -0,0 +1,115 @@
+---
+description: How to install Docker EE for Windows
+keywords: windows, windows server, install, download
+title: Install Docker Enterprise Edition for Windows Server 2016
+---
+
+Docker Enterprise Edition for Windows Server 2016 (*Docker EE*) enables native
+Docker containers on Windows Server 2016. The Docker EE installation package includes everything you need to run Docker on Windows Server 2016 systems. This topic describes pre-install considerations, and how to download and install Docker EE.
+
+> **Already have Docker EE for Windows?** If you already have Docker EE for
+Windows installed, and you're ready to get started, skip to
+[Get started with Docker for Windows](/docker-for-windows/index.md) for a quick tour of the command line, settings, and tools.
+>
+>**Looking for Release Notes?** [Get release notes for all
+versions here](https://docs.docker.com/release-notes/) or subscribe to the [ releases feed on the Docker Blog](http://blog.docker.com/category/engineering/docker-releases/).
+
+## Install Docker EE
+
+Docker EE for Windows requires Windows Server 2016. Please see [What to know before you install](/docker-ee-for-windows/#what-to-know-before-you-install) for a full list of prerequisites. To install Docker Community Edition (*Docker CE*) on a Windows 10 machine, see [Install Docker for Windows](/docker-for-windows/install.md).
+
+1.  Open a PowerShell command prompt, and type the following commands.
+
+    ```bash
+    Install-Module -Name DockerMsftProvider -Force
+    Install-Package -Name docker -ProviderName DockerMsftProvider -Force
+    Restart-Computer -Force
+    ```
+
+2.  Test your Docker EE installation by running the `hello-world` container.
+
+    ```bash
+    docker run hello-world:nanoserver
+    ```
+
+## Install Docker EE on an offline machine
+
+Use the following steps when you want to install Docker EE on a machine that
+isn't connected to the internet.
+
+1.  In a PowerShell command prompt, download the installer archive on a machine that has a connection.
+
+    ```bash
+    # on online machine
+    invoke-webrequest -UseBasicparsing -Outfile docker.zip https://download.docker.com/components/engine/windows-server/17.03/docker-17.03.0-ee.zip
+    ```
+
+2.  Copy the zip file to the machine that isn't connected, by using a thumb drive or by intranet. In a PowerShell command prompt, use the following commands to extract the archive, register, and start the Docker service.
+
+    ```bash
+    #— extract the archive —
+    Expand-Archive docker.zip -DestinationPath $Env:ProgramFiles
+    #clean up zip
+    Remove-Item -Force docker.zip
+    # enable hyper-v if not yet enabled
+    Enable-WindowsOptionalFeature -Online -FeatureName Microsoft-Hyper-V -All
```

Comment:
Is this required? I think it's just this one: https://github.com/OneGet/MicrosoftDockerProvider/blob/developer/DockerMsftProvider.psm1#L595